src/SpectodaRuntime.ts

```
@@ -495,17 +502,17 @@
    * @returns {Function} unbind function
    */
 
-  addEventListener(event: SpectodaTypes.JsEvent, callback: Function) {
+  addEventListener(event: SpectodaJsEventName, callback: Function) {
     return this.on(event, callback);
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)